src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -12,7 +12,7 @@
  * Adds a person to the address book.
  */
 public class AddCommand extends Command {
-
+    public static final boolean type = true;
```

Comment:
Think why you define this variable as above, e.g. I don't think it should be defined as `public`. Also for this particular case, I don't think you need a new constant (you write it as variable though).

Reply:
I thought it is similar to magic number. We should avoid use 'true' directly so I set a new variable